src/components/partnership/dto/partnership.dto.ts

```diff
@@ -0,0 +1,39 @@
+import { GraphQLString } from 'graphql';
+import { Field, ObjectType } from 'type-graphql';
+import {
+  DateTimeField,
+  Resource,
+  SecuredDateTime,
+  SecuredProperty,
+} from '../../../common';
+import { Organization } from '../../organization';
+import { PartnershipAgreementStatus } from './partnership-agreement-status.enum';
+import { PartnershipType } from './partnership-type.enum';
+
+@ObjectType({
+  description: SecuredProperty.descriptionFor('a partnership agreement status'),
+})
+export abstract class SecuredPartnershipAgreementStatus extends SecuredProperty<
+  PartnershipAgreementStatus
+>(GraphQLString) {}
+
+@ObjectType()
+export class Partnership extends Resource {
+  @Field()
+  readonly agreementStatus: SecuredPartnershipAgreementStatus;
+
+  @Field()
+  readonly mouStatus: SecuredPartnershipAgreementStatus;
```

Comment:
Both of these should default to `NotAttached` on partnership create

Reply:
10-4. I'm now setting the default value of `NotAttached` in the partnership's service create method.